fe/fe-core/src/main/java/org/apache/doris/nereids/NereidsPlanner.java

```diff
@@ -536,6 +543,15 @@ private PhysicalPlan chooseBestPlan(Group rootGroup, PhysicalProperties physical
         }
     }
 
+    private long getGarbageCollectionTime() {
+        List<GarbageCollectorMXBean> gcMxBeans = ManagementFactory.getGarbageCollectorMXBeans();
+        long initialGCTime = 0;
+        for (GarbageCollectorMXBean gcBean : gcMxBeans) {
+            initialGCTime += gcBean.getCollectionTime();
+        }
+        return initialGCTime;
+    }
```

Comment:
How long does this function usually take?
If > 10ms, we should only compute this time when SessionVariable.enable_profile == true

Reply:
it should be test on complex condition, it takes 0ms in local test